packages/react-cosmos-component-playground/src/components/LoadersGrid/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class LoadersGrid extends Component {
```

Comment:
Excellent! 

I like `LoadersGrid` more than `LoadersManager`, although I'd opt for `LoaderGrid` (singular) as "grid" already suggests a list of items.